web/themes/custom/votegov/templates/component/usagov-partner.html.twig

```diff
@@ -1,9 +1,11 @@
 <div class="vote-usagov-partner">
   {# USA.gov Links #}
-  <nav class="vote-usagov-partner__links" aria-label={{ "External election resources" | t }}>
-    {# USA.gov election info links #}
-    {{ drupal_entity('block_content', '18') }}
-  </nav>
+  {% if block_18_translations[language] %}
```

Comment:
First set a variable to the loaded block object

`{% set usagov_links_block = drupal_entity('block_content', '18') %}`

To see the variables you have available in twig you can use the `{{ dd(usagov_links_block) }}` function.

Then add a conditional which compares the language and the langcode value from the block

`{% if usagov_links_block['#block_content'].langcode.value == language %}`

Then replace the print statement for the block with the variable you set so we aren't loading it twice.

`{{ usagov_links_block }}`